app/templates/components/build-layout.hbs

```diff
@@ -19,10 +19,17 @@
       {{else}}
         <LoadingIndicator @center={{true}} />
       {{/if}}
-    {{else}}
+    {{else if this.isLog}}
       <JobInfrastructureNotification @job={{this.job}} />
-      <JobTabs @job={{this.job}} @repo={{this.repo}} />
+      <JobTabs
+        @route="build"
+        @model={{this.build}}
```

Comment:
@artursmirnov the purpose of this is to abstract the `JobTabs` component to not have a `build` and `job` attribute, but a `route` and `model` attribute instead. this is because this same component is being used on both `/build/:id` and `/job/:id`. so far, whenever you are on the build route, and click the config tab, you end up on the job route, which can be super confusing.

Reply:
@svenfuchs I totally agree, that ending up on jobs page is confusing, so abstracting the component totally makes sense to me 👍 Thank you for taking this effort! My idea was to have both `@build` and `@job` attributes and use any that's available (e,g, `model: or('job', 'build')` inside the component). In this case the component will be abstract, but the data shape will be clear from both inside and outside of the component. My concern here was that if anyone else will start working on this component in the future, it'll be hard to guess, what `model` is and what data it holds. What do you think?

Reply:
In this case it would even be possible to derive the `route` from the data that's passed. If it's `@job`, then the route is `job`, if it's `@build` then the route is `build`.

Reply:
Here's a little demo of the idea:
```javascript
export default Component.extend({
  build: null,
  job: null,

  model: or('job', 'build'),

  route: computed('job', 'build', function () {
    if (this.job)
      return 'job';
    else if (this.build)
      return 'build';
    else
      throw new Error('You have to pass either job or build to this component')
  })
})
```

Reply:
understood! thanks for the extra explanation ... i've addressed this on epic branch: https://github.com/travis-ci/travis-web/commit/30ef3d543652b6da8836e7b3d5fabd407f8b3349

Reply:
Thank you @svenfuchs, the change looks great! ✨